easybuild/easyconfigs/o/OptiX/OptiX-3.9.0-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,31 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Authors::   Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Binary'
+name = 'OptiX'
+version = '3.9.0'
+
+homepage = 'https://developer.nvidia.com/optix'
+description = """OptiX is NVIDIA SDK for easy ray tracing performance.
+ It provides a simple framework for accessing the GPU’s massive ray tracing
+ power using state-of-the-art GPU algorithms."""
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+
+sources = ['NVIDIA-OptiX-SDK-%(version)s-linux64.sh']
```

Comment:
@thiell no `source_urls`?

If this can't be downloaded automagically, please include a comment with instructions where to download from.